audit/audit.go

```diff
@@ -0,0 +1,64 @@
+package audit
```

Comment:
this package can be extracted to salt later

Reply:
Should we add `TODO` in it @rahmatrhd ?

Reply:
created this issue instead https://github.com/odpf/salt/issues/31